app/views/_user_util_links.html.erb

```diff
@@ -0,0 +1,36 @@
+<ul id="user_utility_links" class="nav navbar-nav navbar-right">
+  <% if user_signed_in? %>
+      <% if current_user.admin? %>
```

Comment:
no, actually these are two separate functionalities.. only admin's should see the link, while all signed in users should see the other stuff as well